servicecontrol/db-compaction.md

```diff
@@ -7,80 +7,39 @@ tags:
 ---
 
 ## Overview
+In order to compact the embedded RavenDB database used by ServiceControl instances, the [Extensible Storage Engine Utility (esentutl)](https://technet.microsoft.com/en-us/library/hh875546.aspx) needs to be run.
```

Comment:
Is it worth adding here something about Raven (/ServiceControl) not recovering disk space once it is consumed?